### DISCUSSION TITLE
Ambient light change with Profile

### DISCUSSION THREAD

**Discussion by mh254** [jilleb/MQB-FPA#46] [Exploration and research]
Hi,

i've coded the Ambient light to change with driving profile, but it did not work.

I flashed a Octavia 3 dataset and it is working.

When i flash back to my Cupra Dataset, it doesn't work anymore.

Any ideas where in the dataset the Ambient light is controlled?

**Comment by jilleb** [comment]
Yes, it's controlled in a similar way to all other controls. Make sure the background color control is in your dataset.

**Reply by mh254** [reply]
But how? Maybe it is to easy so that i oversee something, but right now i wonder what to do?

**Reply by olli991** [reply]
you need to implement that background lighting stuff into the dataset. it is not present in cupra. compare those both ad add those values everywhere @mh254

**Comment by lito69** [comment]
> Hola,
> 
> He codificado la luz ambiental para que cambie con el perfil de conducción, pero no funcionó.
> 
> Mostré un conjunto de datos de Octavia 3 y está funcionando.
> 
> Cuando vuelvo a mi Cupra Dataset, ya no funciona.
> 
> ¿Alguna idea de dónde se controla la luz ambiental en el conjunto de datos?

Rgb? Try this:
[cupra fpa amb.zip](https://github.com/jilleb/MQB-FPA/files/9542177/cupra.fpa.amb.zip)
